### PR TITLE
Replaced deprecated constants, minor fixes

### LINKED ISSUE #36: Replace deprecated constants
I saw this when a user reported issues in #34. These need to be changed ASAP
```
2024-03-30 00:08:35.421 WARNING (MainThread) [homeassistant.const] ELECTRIC_POTENTIAL_VOLT was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricPotential.VOLT instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
2024-03-30 00:08:35.437 WARNING (MainThread) [homeassistant.const] POWER_WATT was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.WATT instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
2024-03-30 00:08:35.453 WARNING (MainThread) [homeassistant.const] ELECTRIC_CURRENT_AMPERE was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricCurrent.AMPERE instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
2024-03-30 00:08:35.469 WARNING (MainThread) [homeassistant.const] ENERGY_KILO_WATT_HOUR was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfEnergy.KILO_WATT_HOUR instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
2024-03-30 00:08:35.484 WARNING (MainThread) [homeassistant.const] POWER_KILO_WATT was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.KILO_WATT instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
2024-03-30 00:08:35.501 WARNING (MainThread) [homeassistant.const] TEMP_CELSIUS was used from nexa_bridge_x, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/andersevenrud/ha-nexa-bridge-x/issues
```

### PR DESCRIPTION
**Pull request by sonite**
Invalid color modes & replace deprecated constants

---

Fixes #36

### CODE REVIEW

File: custom_components/nexa_bridge_x/entities.py [hunk 1]
```diff
@@ -77,8 +77,8 @@ def __init__(self, node: NexaNode, coordinator: CoordinatorEntity) -> None:
 
 class NexaDimmerEntity(NexaNodeEntity, LightEntity):
     """Entity for light"""
-    _attr_color_mode = ColorMode.BRIGHTNESS
-    _attr_supported_color_modes = {ColorMode.ONOFF, ColorMode.BRIGHTNESS}
+    _attr_color_mode = ColorMode.ONOFF
+    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
```
Comment (andersevenrud) [inline custom_components/nexa_bridge_x/entities.py:81]: These changes needs to go into a separate PR and then be verified. Along with dumps of the nodes you have that required these modifications.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Makes it possible to view and control devices set up in the Nexa App/Web UI.
 * Add a custom repository from the hamburger menu
 * Use the [URL](https://github.com/andersevenrud/ha-nexa-bridge-x.git) to this repository and select "integration"
 * Restart HA
+![custom_repos](https://github.com/sonite/ha-nexa-bridge/assets/1986942/ed841f07-1815-41cd-a49c-4ff5dd72c675)
```
Comment (andersevenrud) [inline README.md:33]: Changes to documentation should also be a separate PR.

**Review by andersevenrud — CHANGES_REQUESTED** [review]
Looks good 👍 

But can you separate these ?

File: custom_components/nexa_bridge_x/entities.py (hunk 1 above)
Comment (andersevenrud) [reply custom_components/nexa_bridge_x/entities.py:81]: Ideally an issue should be created if there is some kind of issue or required change to underlying entities, etc.

**Comment by sonite** [comment]
It appears that I'll have to re-fork the repository and make changes file by file, submitting pull requests for each. I can't seem to find an option to edit and split already committed changes. Additionally, GitHub automatically includes all file changes in a commit, so I can't choose which ones to include individually. :(

**Comment by andersevenrud** [comment]
>It appears that I'll have to re-fork the repository and make changes

Not *really*. You can cherry-pick the changes.

> I can't seem to find an option to edit and split already committed changes

Yeah, it's a bit tricky if not familiar with git.

> GitHub automatically includes all file changes in a commit, so I can't choose which ones to include individually. :(

It requires creating branches for each pull-request. The best way to avoid any problems is to not use the "main" branch.

---

If it's too much then I'll find another way to split this up

**Pull request closed without merging**